src/v2/BaseLoginRouter.js

```diff
@@ -259,6 +265,10 @@ export default Router.extend({
     this.listenTo(this.controller, 'all', this.trigger);
 
     this.controller.render();
+
+    this.listenTo(this.controller, 'afterRender', () => {
```

Comment:
See if `triggerAfterRenderEvent` (in FormController) something existing can be helpful ?

Reply:
listening for an event was actually unnecessary